packages/shared/src/components/modals/NewCommentModal.tsx

```diff
@@ -163,6 +165,7 @@ export default function NewCommentModal({
   };
   const useUserMentionOptions = useUserMention({
     postId: props.post.id,
+    sourceId,
```

Comment:
We actually already pass the post right?
Perhaps it's easier to simply pass that if it exists?

Reply:
I think it would be better to minimize the object we're trying to pass as we only need its id (and source now). WDYT?

Reply:
Agree with that, let's go back a step and attribute bare minimum then :)